tests/translator_test.cc

```diff
@@ -338,6 +339,295 @@ TEST(TranslatorTest, TranslateBatchWithPrefixAndEmpty) {
   EXPECT_EQ(result[4].output(), (std::vector<std::string>{"a", "z", "z", "a"}));
 }
 
+TEST(TranslatorTest, TranslateBatchWithStronglyBiasedPrefixAndEmpty) {
```

Comment:
Empty inputs are handled outside of the decoding code, so I don't think it is relevant to include empty inputs when testing this feature.

I suggest removing the empty input and renaming the test case.

Reply:
done